admin/sections/config/model.tsx

```diff
@@ -59,55 +93,57 @@ export const ModelConfig = ({
     };
     // Assign the configured state to the new provider to keep the state (showing the Chat config or not)
     form.reset(newConfig);
-    setValues(newConfig);
   };
 
   return (
     <ExpandableSection
       name="update-model"
-      title={values.configured ? "Update model" : "Start"}
-      description={
-        values.configured
-          ? "Change to a different model or use another provider"
-          : "Set up an AI model to start the app."
-      }
+      title={"Update model"}
```

Comment:
what about showing different text depending on `values.configured`?

Reply:
Ah great catch! i forgot to add this back :D